src/main/java/org/opencraft/server/model/PlayerUI.java

```diff
@@ -219,8 +219,9 @@ public void step(int ticks) {
         String listName = p.getListName();
         if (!listName.equals(playerListName.get(p))){
           playerListName.put(p, listName);
+          short id = (p == this.player ? -1 : p.nameId);
```

Comment:
So instead of sending -1, would it be better to just not send the AddPlayerName at all? For example
```
if (p != this.player) {
          player.getActionSender().sendAddPlayerName(
              p.nameId, p.getName(), p.getListName(), p.getTeamName(), (byte) 1);
        }
}
```
and then you don't have to worry about the -1 constant and whatnot. Or does the -1 have some special meaning?